packages/core/test/tracing/addTracingExtensions.test.ts

```diff
@@ -70,10 +70,5 @@ describe('Tracing extensions', () => {
         }),
       }),
     );
-    expect(spanToJSON(span!)).toEqual(
-      expect.objectContaining({
-        parent_span_id: transaction!.spanContext().spanId,
-      }),
-    );
```

Comment:
We should still check that the span has the correct parent.

````suggestion
    expect(spanToJSON(span!)).toEqual(
      expect.objectContaining({
        parent_span_id: spanToJSON(transaction!).span_id,
      }),
    );
````

Reply:
Got it, seems like the behavior of the JS SDK changed.